pipeline.yml

```diff
@@ -1,43 +1,135 @@
 ---
+groups:
+- name: all
+  jobs:
+  - unit-test-uaa-guard-service
+  - push-uaa-guard-service-staging
+  - push-uaa-guard-service-production
+- name: tests
+  jobs:
+  - unit-test-uaa-guard-service
+  # TODO: Once assertion tests are created
+  #- assertion-tests-uaa-guard-service
```

Comment:
hmm, good point. I completely forgot about how we were moving away from GH issues and just putting everything in Favro. For now, let's just reference #2 in this card and if we get a chance to work on the assertion work we can. For now we'll focus on the PI9 plans we made `anti-fragility`. The base of this work for this PR was to make the `uaa-guard-service` deployment less fragile than it was before when it was artisanal and bespoke. Now that it's automated it's less fragile, but assertion tests would be good to have.